watertap/costing/watertap_costing_package.py

```diff
@@ -734,6 +805,75 @@ def cost_crystallizer_by_volume(blk):
             )
         )
 
+    @staticmethod
+    def compute_steam_properties(blk):

```

Comment:
Should this be private, *or* a helper function in this module?

I mostly think of the static methods on the costing package as being *costing* methods. But this is mostly about how we organize the code.

Reply:
It probably should be a private method (without the static method tag).